java8/scripts.js

```diff
@@ -0,0 +1,17 @@
+document.getElementById('addElem').style.backgroundColor = "rgba(91, 191, 221, 1)";
+document.getElementById('addElem').style.color = "white";
+document.getElementById('addElem').style.borderRadius = "5%";
+document.getElementById('addElem').style.fontSize = "22px";
+document.getElementById('addElem').style.height = "50px";
+document.getElementById('addElem').style.marginLeft = "20px";
+document.getElementById('container').style.display = "flex";
+document.getElementById('container').style.width = "570px";
+
```

Comment:
W zasadzie kolejna uwaga to zamiast dodawać tak style, to lepiej zrobić je wszystkie w css i w javascripcie zrobić tylko `twojElement.classList.add('nazwa-klasy');`